### PR TITLE
ui: add multinode selector

### PR DESCRIPTION
**Pull request by masco**
to select a multiple node press the ctrl key
and select all the nodes you want to select

### CODE REVIEW

File: statics/js/components/node-selector.js [hunk 1]
```diff
@@ -34,6 +34,21 @@ Vue.component('node-selector', {
...
+      });
+
+      $(document).keyup(function(){
+        cntrlIsPressed = false;
+        if (nodes != "") {
+          $(".topology-d3").off('click');
+        }
+      });
+
+      var cntrlIsPressed = false;
```
Comment (hunchback) [inline statics/js/components/node-selector.js]: better to move initialization after line 36

File: statics/js/components/node-selector.js [hunk 2]
```diff
@@ -34,6 +34,21 @@ Vue.component('node-selector', {
...
+      var nodes = "";
```
Comment (hunchback) [inline statics/js/components/node-selector.js]: same as above

Comment (hunchback) [reply statics/js/components/node-selector.js]: may simply the code to make nodes a list:

```
var nodes = [];
```

**Comment by safchain** [comment]
Hi @masco-enovance I made some modification/proposal
https://paste.fedoraproject.org/paste/kzMxyWzZhAn1mLkNgVF5zg

Please have a look and let me know what you think :)

**Comment by lebauce** [comment]
Efforts moved to the new UI

**Pull request closed without merging**